rust/frontend/src/config.rs

```diff
@@ -62,16 +62,23 @@ impl FrontendConfig {
         }
     }
 
-    pub fn single_node_default() -> Self {
+    pub fn single_node_default(url: Option<String>) -> Self {
         // TOOD: unify this with load_from_path to get the env overrides
         let config = DefaultConfigurationsFolder::get(DEFAULT_SINGLE_NODE_CONFIG_FILENAME)
```

Comment:
not sure if this will work when running the built binary outside of the repo folder